lib/diagnostic.rb

```diff
@@ -35,8 +44,9 @@
 # "Somerville". Finally, assign the modified Person to `Response.dave` below.
 
 ## replace nil with your response, then continue your work on the next line
-dave = nil
-Response.dave = dave
+dave = Person.new("Dave", 32, "Ohio")
+dave = Person.location ("Ohio")
+Response.dave = dave.person.update(location: "Ohio")
 ##
```

Comment:
Great job instantiating a new instance of Person!
We can update property values like so once we have our new object
`dave.location = 'Somerville'`